lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -553,19 +555,20 @@ def _create_node(
         if node_context.os_disk_source_file_path:
             # use lock to avoid multiple environments scp disk img to same
             # os_disk_base_file_path.
-            with self._disk_img_copy_lock:
-                source_exists = self.host_node.tools[Ls].path_exists(
-                    path=node_context.os_disk_base_file_path, sudo=True
-                )
-                if source_exists:
-                    self.host_node.tools[Chmod].chmod(
-                        node_context.os_disk_base_file_path, "a+r", sudo=True
-                    )
-                else:
-                    self.host_node.shell.copy(
-                        Path(node_context.os_disk_source_file_path),
-                        Path(node_context.os_disk_base_file_path),
+            with BaseLibvirtPlatform._disk_img_copy_lock:
```

Comment:
Considering that the `host_node` now has a lock, is the `_disk_img_copy_lock` now redundant?

Reply:
`host_node` lock is specific to a platform instance where as `__disk_img_copy_lock` and `_port_forwarding_lock ` are static. These needs to be static as LISA might create multiple instances of platform.